### PR TITLE
Add: website structure and styling

### PR DESCRIPTION
**Pull request by pedrosilva410**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./src/styles.css">
+	<title>Todo List</title>
+</head>
+
+<body>
+	<header>
+		<div class="title-date">
+			<h1>Todo List</h1>
+			<p id="date">date</p> <!--this will display the date-->
```
Comment (201flaviosilva) [inline index.html:15]: There is a tag for that: https://www.w3schools.com/tags/tag_time.asp

I think the `id` it's explicit enough you don't need that comment

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		</div>
+		<div class="input">
+			<input type="text" id="userInput" placeholder="Write down your goals!">
```
Comment (201flaviosilva) [inline index.html:18]: By default, the input it's type text, so you don't need to explicit that

File: src/styles.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  justify-content: space-between;
+  align-items: center;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 60px;
```
Comment (201flaviosilva) [inline src/styles.css:37]: Why?

File: index.html (hunk 1 above)
Comment (pedrosilva410) [reply index.html:15]: 👌

File: index.html (hunk 2 above)
Comment (pedrosilva410) [reply index.html:18]: 👌

File: src/styles.css (hunk 3 above)
Comment (pedrosilva410) [reply src/styles.css:37]: to center it

**Pull request merged**